### PR TITLE
ISSD-1244 [Marketplace] Change from headless admin to delivery in App Marketplace page

### PR DESCRIPTION
**Pull request by juliocezar96**
Subtask : [ISSD-1244](https://liferay.atlassian.net/browse/ISSD-1244)

### CODE REVIEW

**Comment by juliocezar96** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

1 file changed, 70 insertions(+), 34 deletions(-)

**Comment by juliocezar96** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -15,6 +15,12 @@
 		min-width: 3rem;
 	}
 
+	.adt-apps-search-results .card-image-title-container .image-container .app-search-image{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:18]:
```suggestion
	.adt-apps-search-results .card-image-title-container .image-container .app-search-image {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@juliocezar96 can you SF ?/

**Comment by juliocezar96** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 2]
```diff
@@ -57,62 +73,81 @@
...
-													class="h-100 mw-100"
-													src="${srcName}" />
-											</#if>
-										</#list>
-									</#list>
-								</#if>
+								<img
+									alt=${productName}
+									class="app-search-image"
+									src="/o/${productImageParts[1]}"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:108]: What will happen if the productImageParts tobe null or undefined?

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 3]
```diff
@@ -57,62 +73,81 @@
...
 							</div>
 
 							<div class="pl-2">
 								<div class="font-weight-semi-bold h2 mt-1">
-									${product.name.en_US}
+									${productName}
 								</div>
 								<#if productSpecifications?has_content>
-									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
+									<#assign productPriceModelsDeveloperName = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:117]: This name is very strange for me. Could you change it to something like productDeveloperName?
```suggestion
									<#assign productDeveloperName = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl (hunk 2 above)
Comment (juliocezar96) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:108]: Ops. I did the error handling wrong.I will fix this.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 4]
```diff
@@ -57,62 +73,81 @@
...
+							<div class="font-weight-normal mb-2">
+								${productDescription}
 							</div>
+							<#if productSpecifications?has_content>
+								<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "price-model") />
+
+								<#list productPriceModels as productPriceModel>
+									<#if productPriceModel.value?has_content>
+										<#assign priceModel = productPriceModel.value />
+										<#else>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:143]: You need formating this point.
```suggestion
									<#else>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 5]
```diff
@@ -57,62 +73,81 @@
...
 								</div>
 								<#if productSpecifications?has_content>
-									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
+									<#assign productPriceModelsDeveloperName = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
 
-									<#list productPriceModels as productPriceModel>
+									<#list productPriceModelsDeveloperName as productPriceModelDeveloperName>
+										<#if productPriceModelDeveloperName.value?has_content>
+											<#assign developerName = productPriceModelDeveloperName.value />
+											<#else>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:122]: You need to format this point.
```suggestion
										<#else>
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl [hunk 6]
```diff
@@ -49,6 +55,16 @@
 	</div>
 </#if>
 
+<#if themeDisplay?has_content>
+	<#assign scopeGroupId = themeDisplay.getScopeGroupId() />
+</#if>
+
+<#assign channel = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels?accountId=-1&filter=name eq 'Marketplace Channel' and siteGroupId eq '${scopeGroupId}' ") />
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl:62]: Please, remove this white space.
```suggestion
<#assign channel = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels?accountId=-1&filter=name eq 'Marketplace Channel' and siteGroupId eq '${scopeGroupId}'") />
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added more one suggestion

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]
Good job @juliocezar96 Thanks :martial_arts_uniform:

**Comment by silasbsi** [comment]
ci:forward:force

**Pull request closed without merging**